client/src/connection/itc/index.ts

```diff
@@ -333,7 +334,11 @@ export class ITCSession extends Session {
     );
 
     // If we encountered an error in setup, we need to go through everything again
-    const fatalErrors = [/Setup error/, /powershell\.exe/];
+    const fatalErrors = [
+      /Setup error/,
```

Comment:
Not related to this specific change. It looks like we're writing our error strings always as English. Does it make any sense to localize these strings like we do with others in the extension proper?